frontend/app-preview/src/components/AppPreviewSubMenu/AppPreviewSubMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classes from './AppPreviewSubMenu.module.css';
+import { useTranslation } from 'react-i18next';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { PackagesRouter } from 'app-shared/navigation/PackagesRouter';
+import { StudioPageHeaderButton, useMediaQuery } from '@studio/components';
+import { ArrowLeftIcon } from '@studio/icons';
+import { useInstanceIdQuery } from 'app-shared/hooks/queries';
+import { MEDIA_QUERY_MAX_WIDTH } from 'app-shared/constants';
+
+export const AppPreviewSubMenu = () => {
+  const { t } = useTranslation();
+  const shouldDisplayText = !useMediaQuery(MEDIA_QUERY_MAX_WIDTH);
```

Comment:
Out of scope for this PR, but perhaps we should create a StudioMediaQuery component with typed sizes instead of passing a constant directly into it. The useStudioMediaQuery could use useMediaQuery internally? 🤔 

```ts
type StudioMediaQuery = 'small'  | 'medium' | 'large';
const shouldDisplayText = !useStudioMediaQuery('large');
```

Reply:
Would be a great idea for the future 😄 [I've created an issue for it](https://github.com/Altinn/altinn-studio/issues/13662)